svn/common.py

```diff
@@ -22,12 +22,32 @@ def __init__(self, message):
 
 
 class CommonClient(object):
+    """Subversion client
+
+    Parameters
+    ----------
+    url_or_path : str
+        Repository URL or path
+    type_ : str
+        One of ``svn.constants.LT_URL`` or ``svn.constants.LT_PATH``
+    username : str, optional
+        User name to use
+    password : str, optional
+        Password to use
+    svn_filepath : str, optional
+        Path to svn command
+    trust_cert : bool, optional
+        If client should trust server certificate for unknown authority
+    env : dict, optional
+        Environment variables added to ``svn`` process environment
+    """
     def __init__(self, url_or_path, type_, *args, **kwargs):
         self.__url_or_path = url_or_path
         self.__username = kwargs.pop('username', None)
         self.__password = kwargs.pop('password', None)
         self.__svn_filepath = kwargs.pop('svn_filepath', 'svn')
         self.__trust_cert = kwargs.pop('trust_cert', None)
+        self._env = kwargs.get('env', {})
```

Comment:
Also, we don't use the convention of single-underscore prefixed variables, certainly not in all of the other parameters here. You should carry the convention of the surrounding code. Rename `_env` to `__env`.